pptx/dml/color.py

```diff
@@ -91,6 +91,25 @@ def theme_color(self, mso_theme_color_idx):
             self._color = _SchemeColor(schemeClr)
         self._color.theme_color = mso_theme_color_idx
 
+    @property
+    def preset_color(self):
+        """  Preset Color value of this color.
+        
+        Pone of those defined in the MSO_PRESET_COLOR_INDEX enumeration.  
+        None if no preset color is explicitly defined.  Setting this to
+        a value in MSO_PRESET_COLOR_INDEX causes the color's type to 
+        change to ``MSO_COLOR_TYPE.PRESET``
+        """
+        return self._color.preset_color
+
+    @preset_color.setter
+    def preset_color(self, mso_preset_color_val):
+        # change to theme color format if not already
+        if not isinstance(self._color, _PrstColor):
```

Comment:
Can we name it `_PresetColor`? I don't see the need to omit the vowels for only the first word.

Reply:
Is `_PrstColor` an object that steve defined? If it is than it's okay to leave the name. No need to rename and have to hunt it down and change it everywhere else.

Reply:
Unfortunately the naming is blamed mostly on Powerpoint.  the library itself adheres to Powerpoints naming conventions as much as possible so this wasn't my choice.  Fortunately, everything that is externally accessed (properties and such) is proper snake case.